.github/workflows/acceptance-tests.yml

```diff
@@ -56,8 +56,11 @@ jobs:
       - name: Wait for oclif executables Release
         run: source .github/scripts/trigger-and-wait.sh
         env:
-          INPUT_WORKFLOW_FILE_NAME: oclif-release.yml
-          INPUT_REF: ${{ needs.get-branch.outputs.branch }}
```

Comment:
[Q] should we update line:47 since get-branch job is not needed anymore for wait-for-brew-releases job? same question for test-scoop-release job.

Reply:
Yes, good catch.